pip/index.py

```diff
@@ -291,7 +291,10 @@ def _validate_secure_origin(self, logger, location):
                 RemovedInPip7Warning,
             )
 
-    def find_requirement(self, req, upgrade):
+    def _get_indexes_locations(self, req):
```

Comment:
I guess it could be either:

![screen shot 2015-03-10 at 11 31 25 am](https://cloud.githubusercontent.com/assets/305268/6582219/204b95ce-c719-11e4-8c58-c13b31457721.png)

(from http://www.merriam-webster.com/dictionary/index)